### PR TITLE
Issue #563

### PR DESCRIPTION
**Pull request by stepanzharychevbroadcom**

### CODE REVIEW

File: src/utils/workspace.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import * as vscode from "vscode";
+
+interface IExtTextEditor extends vscode.TextEditor {
+    id: string;
+}
+
+export async function checkTextFileIsOpened(path: string) {
```
Comment (VitGottwald) [inline src/utils/workspace.ts]: Are we sure this does not interfere with the user work flow - like blinking a window or activating some other window than currently selected?

Comment (stepanzharychevbroadcom) [reply src/utils/workspace.ts]: It will be, as I showed you, but unfortunately there's no other way for us to achieve the requested functionality.

**Comment by stepanzharychevbroadcom** [comment]
> Thanks Stepan an interesting solution. Needs test coverage please
> image

@Colin-Stone Should be fixed now.

**Comment by stepanzharychevbroadcom** [comment]
@Colin-Stone @VitGottwald @zFernand0 
What deserves special attention in this review: **workspace.ts**.
I introduced the fix which omits empty/binary tabs up to 3 times, and after stops. (this condition required not to go to the infinite loop) It's reflected in tests.

Also I added **constants**, so magic values can be used across the application, not stay hardcoded in 1 file.

**Comment by stepanzharychevbroadcom** [comment]
New implementation is in https://github.com/zowe/vscode-extension-for-zowe/pull/817. PR should be closed.

**Pull request closed without merging**